rs/rosetta-api/icp/tests/system_tests/common/governance_client.rs

```diff
@@ -0,0 +1,107 @@
+use candid::{Decode, Encode, Principal};
+use ic_agent::Agent;
+use ic_base_types::PrincipalId;
+use ic_nns_common::{pb::v1::ProposalId, types::NeuronId};
+use ic_nns_governance_api::pb::v1::Motion;
+use ic_nns_governance_api::pb::v1::ProposalActionRequest;
+use ic_nns_governance_api::{
+    pb::v1::{
+        manage_neuron_response, manage_neuron_response::MakeProposalResponse, MakeProposalRequest,
+        ManageNeuronResponse, ProposalInfo,
+    },
+    proposal_submission_helpers::create_make_proposal_payload,
+};
+
+pub struct GovernanceClient {
```

Comment:
What would the disadvantages of my approach be?

Putting helper functions in a contextualized entity (like the `GovernanceClient` class in this case) is usually considered a better practice as it makes it easier to find and understand utility methods instead of having it mixed together with utilities that do distinct stuff. Having it be stateful allow us to have to repeat using the same method arguments over and over again and puts the methods in a contextualized namespace.